application/templates/prometheus-rule.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.prometheusRule -}}
+{{- if .Values.prometheusRule.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+{{ toYaml .Values.prometheusRule.labels | indent 4 }}
```

Comment:
what if there are no labels, it will fail since there is no if condition that check for label existence